src/Objs/Job/JobComponent.js

```diff
@@ -17,14 +17,20 @@ Scrivito.provideComponent("Job", ({ page }) => {
                 content={page}
                 attribute="title"
               />
-              <JobDatePosted datePosted={page.get("datePosted")} />
+              <Scrivito.ContentTag
+                content={page}
+                attribute="datePosted"
+                tag="span"
+              >
+                <JobDatePosted datePosted={page.get("datePosted")} />
+              </Scrivito.ContentTag>
             </div>
             <div className="col-lg-5 details-title-box">
               <JobLocation job={page} />
               <JobEmploymentTypes
                 employmentTypes={page.get("employmentType")}
               />
-              <JobValidThrough validThrough={page.get("validThrough")} />
+              <JobValidThrough page={page} />
```

Comment:
* [ ] The level of abstraction is not consistent. `JobDatePosted` is wrapped by `Scrivito.ContentTag`, `JobValidThrough` wraps `Scrivito.ContentTag`. Can we make it one or the other?